docs/en/toolkit/toolkit_basics_guides/advanced_config.md

```diff
@@ -129,7 +129,7 @@ A dialog box is displayed with the name of the project populating the text field
 
 ![Project folder](../../../images/learning_content/toolkit_basics_guides/advanced_config/14_project_folder_name.png)
 
-Toolkit can work in either a Distributed Setup, where the pipeline configuration is uploaded to Shotgun and cached locally for each user, or a Centralized Setup, where users access a single configuration in a shared location on disk. For this exercise we will use a Centralized Setup. You can [learn more about Distributed Setups here](https://developer.shotgunsoftware.com/tk-core/initializing.html#distributed-configurations).
+Toolkit can work in either a Distributed Setup, where the pipeline configuration is uploaded to Shotgun and cached locally for each user or a Centralized Setup, where users access a single configuration in a shared location on disk. For this exercise we will use a Centralized Setup. You can [learn more about Distributed Setups here](https://developer.shotgunsoftware.com/tk-core/initializing.html#distributed-configurations).
 
```

Comment:
Sorry, one more note!

I'd like to keep this comma, otherwise it's unclear where the clause that starts with "where the pipeline configuration.." ends.

Reply:
OK, but the "or" is what created the separation... it's not correct english... I think it's clear without the comma.